probe_scraper/scrapers/git_scraper.py

```diff
@@ -95,51 +94,50 @@ def retrieve_files(repo_info, cache_dir):
     results = defaultdict(list)
     timestamps = dict()
     base_path = os.path.join(cache_dir, repo_info.name)
+    repo_path = f"{base_path}.git"
```

Comment:
oooh, so before we were mixing the git checkout and some file checkouts into the same directory, making it harder to cache one but not the other. That makes sense now!